src/viam/sdk/common/utils.hpp

```diff
@@ -89,5 +89,8 @@ void set_logger_severity_from_args(int argc, char** argv);
 /// @param extra The extra AttributeMap.
 bool from_dm_from_extra(const AttributeMap& extra);
 
+/// helper to throw on !ok grpc::Status
+#define THROW_NOT_OK(status) if (!status.ok()) { throw std::runtime_error(status.error_message()); }
```

Comment:
Somehow my comment didn't make it onto the review:

> Let's not put this in a common location just yet. We have no facilities to ensure macro hygiene right now. I'd rather address this as part of the (I believe upcoming) work to refactor error handling and exceptions throughout the codebase. Fine to push it down into the file that uses it for now.

Reply:
removed wrapper for now 8673dc7